src/swift/Sources/Models/TokenFile.swift

```diff
@@ -229,465 +234,430 @@ class TokenFile: Codable {
     }
 
-    private func process(_ decl : InitializerDeclaration) {
-        SharedLogger.debug("Initializer Declaration")
-        guard publicModifiers.contains(decl.modifiers.accessLevel ?? .internal) else {
-            return
-        }
-        newLine()
-        if !decl.attributes.isEmpty {
-            handle(attributes: decl.attributes)
-        }
-        if !decl.modifiers.isEmpty {
-            handle(modifiers: decl.modifiers)
-        }
+    private func process(_ decl: InitializerDeclaration, overridingAccess: AccessLevelModifier? = nil) -> Bool {
+        let accessLevel = decl.modifiers.accessLevel ?? overridingAccess ?? .internal
         let defId = decl.textDescription
-        keyword(value: "init")
-        lineIdMarker(definitionId: defId)
-        if let genericParam = decl.genericParameterClause {
-            handle(clause: genericParam)
-        }
-        let initSignature = FunctionSignature(parameterList: decl.parameterList, throwsKind: decl.throwsKind, result: nil)
-        handle(signature: initSignature)
-        if let genericWhere = decl.genericWhereClause {
-            handle(clause: genericWhere)
-        }
+        return processInitializer(defId: defId, attributes: decl.attributes, modifiers: decl.modifiers, kind: decl.kind.textDescription, accessLevel: accessLevel,  genericParam: decl.genericParameterClause, throwsKind: decl.throwsKind, parameterList: decl.parameterList, genericWhere: decl.genericWhereClause)
     }
 
-    private func process(_ decl: FunctionDeclaration) {
-        SharedLogger.debug("Function Declaration")
-        SharedLogger.debug(decl.signature.textDescription)
+    private func process(_ decl: FunctionDeclaration, overridingAccess: AccessLevelModifier? = nil) -> Bool {
+        let accessLevel = decl.modifiers.accessLevel ?? overridingAccess ?? .internal
+        let defId = decl.textDescription
+        let name = decl.name.textDescription
+        return processFunction(name: name, defId: defId, attributes: decl.attributes, modifiers: decl.modifiers, accessLevel: accessLevel, signature: decl.signature, genericParam: decl.genericParameterClause, genericWhere: decl.genericWhereClause)
+    }
 
-        guard publicModifiers.contains(decl.modifiers.accessLevel ?? .internal) else {
-            return
+    private func process(_ decl: SubscriptDeclaration, overridingAccess: AccessLevelModifier? = nil) -> Bool {
+        let accessLevel = decl.modifiers.accessLevel ?? overridingAccess ?? .internal
+        let defId = decl.textDescription
+        return processSubscript(defId: defId, attributes: decl.attributes, modifiers: decl.modifiers, accessLevel: accessLevel, genericParam: decl.genericParameterClause, parameterList: decl.parameterList, resultType: decl.resultType, genericWhere: decl.genericWhereClause)
+    }
+
+    private func process(_ decl: OperatorDeclaration) -> Bool {
```

Comment:
Why doesn't this one have `overridingAccess`?

Reply:
Operators are always global and never have access modifiers. They can only be `public`.